apps/frontend/src/libs/hooks/use-app-form/use-app-form.hook.ts

```diff
@@ -43,13 +45,15 @@ const useAppForm = <T extends FieldValues = FieldValues>({
 		control,
 		formState: { errors, isDirty },
 		handleSubmit,
+		watch,
```

Comment:
we are already using useWatch in the project, let's stick to one approach
```suggestion
```

Reply:
Oh, I didn't notice that, thanks